src/generic/generic_arithmetic.h

```diff
@@ -158,14 +184,14 @@ static inline void mayo_generic_m_calculate_PS(const uint64_t *P1, const uint64_
     // multiply stuff according to the bins of the accumulator and add to PS.
     int i = 0;
     while (i < n * k) {
-        m_vec_multiply_bins(m_vec_limbs, accumulator + i * 16 * m_vec_limbs, PS + i * m_vec_limbs);
+        m_vec_multiply_bins(m_vec_limbs, accumulator + i * 16 * m_vec_limbs, SPS + i * m_vec_limbs);
```

Comment:
What is this change for? The function. is still computing PS, not SPS, right? Why rename the buffer to SPS?

Reply:
Thanks for the catch @mkannwischer, no idea why I changed the variable name.. fixed now.

Reply:
thx